src/formatting_stack/linters/eastwood.clj

```diff
@@ -28,6 +32,14 @@
 (note that this prefix must not be passed to Eastwood itself).")
 
 (defn lint! [{:keys [options]} filenames]
+  {:post [(do
```

Comment:
A glorified :such-that ;p

An alternative (or complementary) solution would be strengthening `::protocols.spec/filename` from `present-string?` to a string that denotes an absolutized, existing filename.

IIRC a very similar idea was rejected ~1y ago (can't find the link, sorry) so I'm fine with leaving things as-is

Reply:
> from present-string? to a string that denotes an absolutized, existing filename. IIRC a very similar idea was rejected ~1y ago 

I guess the reasoning would be something akin to "don't have side-effecting specs". I.e. a absolute filepath is fine (i.e. leading `/`), going through the filesystem is not.

It makes generating examples harder and makes a spec unpure / dependent on state outside of it's input